backend/prisma/schema.prisma

```diff
@@ -7,28 +7,12 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-// model Post {
-//   id        Int      @id @default(autoincrement())
-//   createdAt DateTime @default(now())
-//   updatedAt DateTime @updatedAt
-//   title     String   @db.VarChar(255)
-//   content   String?
-//   published Boolean  @default(false)
-//   author    User     @relation(fields: [authorId], references: [id])
-//   authorId  Int
-// }
-
-// model Profile {
-//   id     Int     @id @default(autoincrement())
-//   bio    String?
-//   user   User    @relation(fields: [userId], references: [id])
-//   userId Int     @unique
-// }
+model User {
+  id      Int      @id @default(autoincrement())
+  email   String   @unique
+  firstName    String 
+  lastName String 
+  passwords String 
```

Comment:
A user probably will only have one password here, so this field is probably just `password`

Reply:
okay i fixed the passwords -> password